.gitignore

```diff
@@ -26,6 +26,7 @@ database.yml
 
 # Ignore coverage results
 /coverage
+/config/application.yml
```

Comment:
alright i will update it, i actually deleted my application.yml but i will delete this line of ignoring it